builtin/providers/aws/provider.go

```diff
@@ -214,6 +214,9 @@ func Provider() terraform.ResourceProvider {
 			"aws_db_parameter_group":                       resourceAwsDbParameterGroup(),
 			"aws_db_security_group":                        resourceAwsDbSecurityGroup(),
 			"aws_db_subnet_group":                          resourceAwsDbSubnetGroup(),
+			"aws_dc_virtual_interface":                     resourceAwsDirectConnectVirtualInterface(),
+			"aws_dc_intra_virtual_interface":               resourceAwsDirectConnectIntraVirtualInterface(),
+			"aws_dc_intra_virtual_interface_confirm":       resourceAwsDirectConnectIntraVirtualInterfaceConfirm(),
```

Comment:
AWS refer to direct connects as 'dx' is a fair amount of their documentation (see: https://aws.amazon.com/directconnect/faqs/)